core/eth/tx.go

```diff
@@ -370,9 +370,11 @@ func (t *Transactor) GetOperatorStakes(ctx context.Context, operator core.Operat
 		return nil, nil, err
 	}
 
+	quorumIds := BitmapToQuorumIds(quorumBitmap)
+
 	state := make(core.OperatorStakes, len(state_))
 	for i := range state_ {
-		quorumID := core.QuorumID(i)
+		quorumID := quorumIds[i]
```

Comment:
So we're assuming `BitmapToQuorumIds` returned an ordered list of quorums in ascending order, and the quorums in `state_` is in the same order, which is true in the [current implementation](https://github.com/Layr-Labs/eigenlayer-middleware/blob/dev/src/OperatorStateRetriever.sol#L40).